geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -211,7 +231,16 @@ trait HBaseFeatureIndex extends HBaseFeatureIndexType
       val remoteFilters = if (remote) { ecql.map { filter =>
         new JSimpleFeatureFilter(sft, filter)
       }.toSeq } else { Nil }
-      ScanConfig(remoteFilters, toFeatures)
+
+      val coprocessor: Option[Coprocessor] = {
+        if (hints.isDensityQuery) {
+          Some(new KryoLazyDensityCoprocessor)
```

Comment:
Basically.

```
        if(ds.config.isBigtable) {
          bigtableScanPlan(ds, filter, ranges, table, hbaseFilters, toFeatures)
        } else {
          hbaseScanPlan(sft, filter, ranges, table, hbaseFilters, coprocessor, hints, toFeatures)
        }
```

This will prevent it from executing coprocessor code on big table.